app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java

```diff
@@ -59,6 +62,11 @@ protected void onCreate(Bundle savedInstanceState) {
 
         toolbar = (Toolbar)findViewById(R.id.toolbar_trustees);
         setSupportActionBar(toolbar);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            getWindow().addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS);
+            getWindow().clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
+            getWindow().setStatusBarColor(ContextCompat.getColor(this, R.color.status_bar_color));
+        }
```

Comment:
The same code duplicates in two activities. Can you create a method in a util class and move the code segment to there. Then refer the newly created method in both places.